### ISSUE TITLE
Identify “friendlies” to pilot the MOC to NERC migration doc

### ISSUE THREAD

**Issue by joachimweyl** [CCI-MOC/ops-issues#495]
- [x] @joachimweyl track down form/survey that requested users to decide if they are interested in migrating from MOC to NERC
- [x] @joachimweyl work with Milson to get "friendlies" to test the migration
- [x] blocked on #596 
- [x] @joachimweyl have at least 2 "friendlies" test
    - [x] FPGA - blocked on #600 
    - [x] SAIL (starting, but we are gonna send mail anyway)
- [x] @joachimweyl send it out again to get an up to date response once a migration date is committed

**Comment by joachimweyl** [comment]
- [ ] @joachimweyl track down form/survey that requested users to decide if they are interested in migrating from MOC to NERC
- [ ] @joachimweyl send it out again to get an up to date response
- [ ] @joachimweyl provide a current list of projects that want to migrate to the NERC based on indicated desire to move to the NERC in response to the survey.

**Comment by msdisme** [comment]
SAIL projects initially, any others from scott/wayne

**Comment by joachimweyl** [comment]
After discussion with @msdisme, we are waiting for NERC to say they are ready (target for June).

**Comment by joachimweyl** [comment]
FPGA ran into an issue while testing.

**Comment by msdisme** [comment]
Query with Milson - a. can we do this sooner (blocked on FPGA nd ned to put up flavors, b. should fpga testing be a blocker

**Comment by msdisme** [comment]
unblocked for smaller projects.  Send mail.

**Comment by joachimweyl** [comment]
As this issue was just to identify the friendlies we have already done this. I will close this and let #535 manage the rest of this process.
